### PR TITLE
Add labels to the IconButtons in UrlInput

### PR DESCRIPTION
**Pull request by prtksxna**

### CODE REVIEW

**Comment by prtksxna** [comment]
This addresses #1855 and #1854.

File: blocks/url-input/index.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ class UrlInput extends Component {
 			<li className="components-url-input">
 				<IconButton
 					icon="admin-links"
+					label={ __( 'Link' ) }
```
Comment (mtias) [inline blocks/url-input/index.js]: This one looks like it could be a little more clear. Maybe "Edit Image Link" or similar. cc @afercia

Comment (prtksxna) [inline blocks/url-input/index.js]: Makes sense 👍

**Comment by afercia** [comment]
Thanks @prtksxna! I'd consider to make a bit more clear also the other ones:
- "Back": sounds like page navigation; what this control does, is it closes the link "toolbar" and switches back to the default toolbar, not sure how to communicate this in a short wording; maybe just "Close"? /cc @mtias 
- `aria-label="Enter URL" placeholder="Paste URL or type"`: "Enter URL" in combination with the placeholder is a bit repetitive, I'd use just "URL"
- the "Apply" button still misses a label 😉 

@mtias on a side note: when a link is inserted, the behavior is different from text links (they have an "unlink" button): ideally, this should be consistent with the controls for text links

**Comment by mtias** [comment]
"Close" sounds good to me.

> when a link is inserted, the behavior is different from text links (they have an "unlink" button): ideally, this should be consistent with the controls for text links.

The interactions are a bit different than text links. Explored it a bit with @joen because, in this case, it's more like an attribute of the block (have the image point to somewhere) for which we didn't have proper patterns yet.

**Comment by prtksxna** [comment]
Thanks for the review @afercia and @mtias !

> the "Apply" button still misses a label 😉

Right. I saw the `type="submit"` there and assumed that it would translate into an `<input>` with that attribute, in which case that would have been enough for the screen reader to understand its purpose.

But, as it becomes a `<button>` I am not so sure about it. Will update this.

> The interactions are a bit different than text links. Explored it a bit with @joen because, in this case, it's more like an attribute of the block (have the image point to somewhere) for which we didn't have proper patterns yet.

So is it alright to leave it as such for now? You'd be adding the _unlink_ button (or not) in a separate patch?

**Pull request merged**